File: U3DEventFrame/Assets/DemoScript/UIDemo/UIEvents/UIDemoEvents.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

using  U3DEventFrame ;


public enum UIDemoEvents
{

    Initial =  ManagerID.UIManager +1 ,



    DemoMsg,

    MaxValue 
}


public enum UIDemoLoading

{
 
     Initial =  UIDemoEvents.MaxValue +1 ,

     GetResources ,



      ShowPanel  ,

     MaxValue

}


public enum UIDemoRegisting
{

    Initial = UIDemoLoading.MaxValue + 1,

    GetResources,

    

    MaxValue

}

public enum UIDemoLoadingEvents
{
    Initial =ManagerID.LUIManager+2,

    GetResources,

    TestMsg,

    MaxValue

}

 

File: U3DEventFrame/Assets/Lua/Demo/UIDemoEvents.lua
--region *.lua
--Date
--此文件由[BabeLua]插件自动生成



--endregion




local    CUIDemoMsgBegin  =   LManagerID.UIManager +1


CUIDemoMsgEvent = {


"DemoMsg",

DemoMsg


}


ResetTableKeyValue(CUIDemoMsgBegin,CUIDemoMsgEvent);


local   uiEventsBegin =  LManagerID.LUIManager +1  ;


UIDemoLoadingEvents = {


   "Initial",

   "GetResources",

   "TestMsg",

   "MaxValue",


   Initial,

   GetResources ,

   TestMsg,

   MaxValue


}

ResetTableKeyValue(uiEventsBegin,UIDemoLoadingEvents);



UIDemoRegistEvents = {

    "Initial",

   "GetResources",

     

    "MaxValue",

    Initial,

    GetResources ,

    MaxValue


}

ResetTableKeyValue(UIDemoLoadingEvents.MaxValue,UIDemoRegistEvents);



LRoateSelfEevent = {

    "Initial",

   "GetResources",

     

    "MaxValue",

    Initial,

    GetResources ,

    MaxValue


}

ResetTableKeyValue(UIDemoRegistEvents.MaxValue,LRoateSelfEevent);

